pkg/minikube/localpath/localpath.go

```diff
@@ -32,7 +32,9 @@ import (
 const MinikubeHome = "MINIKUBE_HOME"
 
 // ConfigFile is the path of the config file
-var ConfigFile = MakeMiniPath("config", "config.json")
+func ConfigFile() string {
```

Comment:
Change this package variable into func(). Otherwise, setting MinikubeHome env in test cases will not have any impact.